### DISCUSSION TITLE
Post Info Module - Allow number of taxonomy terms limit

### DISCUSSION THREAD

**Discussion by pixelloop** [elementor/elementor#22508] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Something I have used with other page builders and I have done with PHP filters before but can't find any way to implement with Elementor is a limit on the number of taxonomy terms to show when using the post info widget.

The post info widget allows you to display all categories or custom taxonomy terms for a single post, but in some cases many are added but we don't want to display a long list.

Having the ability to limit this to show 1 or 2 categories, tags or custom taxonomies would be very useful. 

### Describe the solution you'd like

On the post info widget, when categories or taxonomies are added, allow for an input box where you can add a custom number of taxonomy terms to show with the title above "Limit number of taxonomy terms to display". 

### Describe alternatives you've considered

Custom PHP function/filter on "get_the_terms" unfortunately, any of the filters I have used in the past don't seem to work with Elementor's method for outputting the terms.

**Comment by henry-parkside** [comment]
I am also looking for this exact feature today! Hopefully this gets seen by Elementor...
